src/System.Reflection.TypeExtensions/src/System/Reflection/TypeExtensions.cs

```diff
@@ -177,25 +177,25 @@ public static PropertyInfo[] GetProperties(Type type, BindingFlags bindingAttr)
             return type.GetProperties(bindingAttr);
         }
 
-        public static PropertyInfo GetProperty(Type type, string name)
+        public static PropertyInfo? GetProperty(Type type, string name)
         {
             Requires.NotNull(type, nameof(type));
             return type.GetProperty(name);
         }
 
-        public static PropertyInfo GetProperty(Type type, string name, BindingFlags bindingAttr)
+        public static PropertyInfo? GetProperty(Type type, string name, BindingFlags bindingAttr)
         {
             Requires.NotNull(type, nameof(type));
             return type.GetProperty(name, bindingAttr);
         }
 
-        public static PropertyInfo GetProperty(Type type, string name, Type returnType)
+        public static PropertyInfo? GetProperty(Type type, string name, Type returnType)
```

Comment:
It seems like `returnType` should be nullable. https://source.dot.net/#System.Private.CoreLib/shared/System/Type.cs,224